ironfish/src/mining/manager.ts

```diff
@@ -95,6 +104,13 @@ export class MiningManager {
         nullifiers.add(spend.nullifier)
       }
 
+      // Stop adding transactions when the addition would cause the block to exceed the max size
+      const transactionSize = getTransactionSize(transaction.serialize())
```

Comment:
This will abort and not add any transactions in the current mem pool if an extremely large TX is generated that exceeds max block size with an also large fee. An example of this is when we do pool payouts. https://explorer.ironfish.network/transaction/becc7ff76a29248150bb99b620ccbfdeae6c0be1e31a1d6910c661c0a744b340